samples/js/build.gradle.kts

```diff
@@ -28,6 +30,12 @@ version = "0.0.1"
 
 kotlin {
     js {
-        nodejs()
+        nodejs {
+            testTask {
+                testLogging {
+                    exceptionFormat = TestExceptionFormat.FULL // Show full exception when an assertion fails
```

Comment:
We could also add in the following options for explanation sake and to be consistent with the other sample `build.gradle.kts`.

```
    testLogging {
        exceptionFormat = TestExceptionFormat.FULL
        showExceptions = true        // defaults to true
        showCauses = true            // defaults to true
        showStackTraces = true       // defaults to true
    }
```

Reply:
good idea, let's do this